### DISCUSSION TITLE
MLFLOW Authentication via OKTA

### DISCUSSION THREAD

**Discussion by fabiosanger** [mlflow/mlflow#12009] [Q&A]
I have been tasked to add OKTA authentication to the mlflow tracking server. I got already the app set up on the internal OKTA server and therefore a Client ID/Secret.
is the basic_auth.db on the tracking server?
How would i add the authentication to the server?

**Comment by idrissa-mgs** [comment]
Hello @fabiosanger did you find the solution finally. I come across this https://github.com/mlflow-oidc/mlflow-oidc-auth?tab=readme-ov-file but not tested yet

**Reply by fabiogeraci** [reply]
yes the issue was on the OKTA groups defintion
